frontend/app/_clients/champions-client/champions-client-type.d.ts

```diff
@@ -0,0 +1,7 @@
+type ChampionsClient = {
+  champions: ChampionData[];
+  championSearchValue: string;
+  setChampionSearchValue: import("react").Dispatch<
+    import("react").SetStateAction<string>
+  >;
```

Comment:
best to simplify this type like:
```suggestion
  setChampionSearchValue: (searchTerm: string)=>void;
```